app/components/dragable-elements.js

```diff
@@ -0,0 +1,80 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    didInsertElement: function() {
+        var self = this;
+        var hopperOutlet = this.$().closest('.hopper-outlet');
+        hopperOutlet.find('.sub-elements').sortable({
+            connectWith: ".sub-elements",
+            cursor: "move",
+            cancel: "input,textarea,a",
+            delay: 150,
+            forceHelperSize: true,
+            forcePlaceholderSize: true,
+            toleranceType: "pointer",
+            update: function(event, ui) {
+                if (ui.item.find('.field').hasClass('fieldset') &&
+                        ui.item.parents().eq(2).hasClass('fieldset')) {
+                    hopperOutlet.find('.sub-elements').sortable('cancel');
+                }
+                var order = {};
+                Ember.$('.field').each(function(index) {
+                    var element = Ember.$(this);
+                    var parentNode = element.parents().eq(3);
```

Comment:
$().parents() returns all elements upwards till the root node, starting with the current one. The element 0 is the current one. 1 is the div.sub-element we are using to connect with. 2 is a div, added by ember. Therefore, getting the hopper-outlet or the parent element we need to use the 3.